setup.py

```diff
@@ -401,6 +407,16 @@ def parse_jsargs():
 
     return jsbuild
 
+
+def package_tree(pkgroot):
+    """ Get list of packages by walking the directory structure and
+    including all dirs that have an __init__.py or are named test.
+    """
+    subdirs = [os.path.relpath(i[0], ROOT).replace(os.path.sep, '.')
+               for i in os.walk(os.path.join(ROOT, pkgroot))
+               if '__init__.py' in i[2] or os.path.split(i[0])[1] == 'tests']
```

Comment:
I think we should just put `__init__.py` in all the `test directories. There was some talk about it having issues with `py.test` but lots of them already do, and it seems to have no ill effect at all. ping @birdsarah